src/utils/utils.ts

```diff
@@ -95,20 +95,30 @@ function outputFormatter(msg: OutputObject): string | Uint8Array {
       output = output.substring(0, output.length - 1) + `\n`;
     }
   } else if (msg.type === 'dict') {
+    // Step 1: Find the length of the longest key
+    let maxKeyLength = 0;
+    for (const key in msg.data) {
+      if (key.length > maxKeyLength) {
+        maxKeyLength = key.length;
+      }
+    }
+
+    // Step 2: Use the maxKeyLength to align the keys and values
     for (const key in msg.data) {
       let value = msg.data[key];
       // Empty string for null or undefined values
       if (value == null) {
         value = '';
       }
-      value = JSON.stringify(value);
+      value = JSON.stringify(value).replace(/"/g, '');
```

Comment:
This isn't safe. There may be actual `"` inside the values of the JSON. It's better to left and right trim the `"` if the value is of type string.

Reply:
I modified the code to only remove the starting and ending quotes if the value is of type string.